README.md

```diff
@@ -52,10 +52,14 @@ Copy sample environment file and configure it as per your local settings
 cp .env.example .env
 ```
 
-Install dependencies
+Setup virtual environment and install dependencies
 
 ```
-pip3 install -r requirements.txt
+pip3 install virtualenv
+virtualenv venv
+source venv/bin/actiavte
```

Comment:
Typo :) 
Also please add the `deactivate` command as well

Reply:
#26 does the same

Reply:
typo